xrpackage-api.js

```diff
@@ -0,0 +1,554 @@
+import * as THREE from './xrpackage/three.module';
```

Comment:
Do these imports need a `.js` extension? I'm getting errors like:
```
GET http://localhost:3000/xrpackage/GLTFLoader net::ERR_ABORTED 404 xrpackage-api.js:4
GET http://localhost:3000/xrpackage/VOXLoader net::ERR_ABORTED 404
```
locally.

I feel like it should work if the express server didn't care about the extensions but I think github pages might care about them when it's deployed (unless we used something like webpack maybe?)

Reply:
That's strange! I didn't need the extensions before, but you are indeed correct that the errors disappear when I add the extension

Reply:
I think it's required when running on the web. We don't use a bundler.